charts/springboot/values.yaml

```diff
@@ -12,6 +12,10 @@ service:
   internalPort: 8080
 ingress:
   enabled: false
+# Manages external ingress. Needs to specify separate ingressClass to work properly.  
+ingressExternal:
+  enabled: false
+  # ingressClass: "kong-external"
```

Comment:
Pas mus ūkyje jau ir istio yra, todėl siūlau šitą daryti kong specific ir atitinkamai pavadinti

Reply:
Turi galvoje ingressExternal į pvz. kongExternal?  Čia mintis buvo, kad jei yra keli ingress kontroleriai, būtų galima nurodyt jei reikia.

Reply:
Nepastebėjau, kad turi if'us ant `.Values.ingressExternal.kong` Tai ok tada